lib/shared/addon/components/cluster-driver/driver-azureaks/template.hbs

```diff
@@ -136,15 +130,74 @@
           </div>
           <div class="col span-6">
             <label class="acc-label" for="azureaks-master-dns-prefix">{{t 'clusterNew.azureaks.dns.label'}}</label>
-            {{input
-                classNames="form-control"
-                id="azureaks-master-dns-prefix"
-                placeholder=(t 'clusterNew.azureaks.dns.placeholder')
-                type="text"
-                value=config.masterDnsPrefix
+            {{#input-or-display
+                 editable=isNew
+                 value=config.masterDnsPrefix
+            }}
+              {{input
+                  classNames="form-control"
+                  id="azureaks-master-dns-prefix"
+                  placeholder=(t 'clusterNew.azureaks.dns.placeholder')
+                  type="text"
+                  value=config.masterDnsPrefix
+              }}
+            {{/input-or-display}}
+          </div>
+        </div>
+
+        <div class="row">
+          <div class="col span-6">
+            <label class="acc-label">{{t 'clusterNew.azureaks.enableHttpApplicationRouting.label'}}</label>
```

Comment:
We're mixing single and double quotes in the same html file. We dont have a template linter yet (coming in the next ember upgrade) but when we do the standard is double qoutes in HBS files.

If you are working on an HBS file I am okay if you do a global (file global) find and replace of single quotes to double quotes.